sklearn/feature_selection/_rfe.py

```diff
@@ -108,11 +103,16 @@ class RFE(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
         class:`~sklearn.pipeline.Pipeline` with its last step named `clf`.
 
         If `callable`, overrides the default feature importance getter.
-        The callable is passed with the fitted estimator and it should
-        return importance for each feature.
+        The callable is passed with the fitted estimator and optionally the
+        training input samples `X` and the target labels `y`. The callable
+        should return importance for each feature.
 
         .. versionadded:: 0.24
 
+        .. versionchanged:: 1.2
+           Added support for custom importance getter with estimator, training input
+           samples `X` and the target labels `y`.
```

Comment:
I feel like we could improve wording here. What do you think? (May need to format w/ black)

```suggestion
           Added support for a callable `importance_getter` which accepts estimator, training input
           samples `X` and the target labels `y` as arguments.
```